bedrock/externalpages/templates/externalpages/pocket/jobs.html

```diff
@@ -0,0 +1,70 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "externalpages/pocket/base.html" %}
+
+{% from "macros-protocol.html" import split with context %}
+
+{% block pocket_css %}
+{{ css_bundle('pocket-jobs') }}
+{% endblock %}
+
+{% block page_title %}Jobs{% endblock page_title %}
+
+{% block content %}
+  {% include 'externalpages/pocket/includes/nav.html' %}
+  <div class="mzp-l-content mzp-t-content-xl pocket-jobs-wrapper">
+    <h1>Join the Pocket Team</h1>
+    <figure>
+      {{ high_res_img('img/externalpages/pocket/jobs/jobs-team.jpg', {'alt': '', 'width': '745', 'height': '408'}) }}
```

Comment:
Why not use the `static` method for images and use the `srcset` string for high-res images? It would be nicer to implement that to help with performance for some users. Similar code could be seen in [this internships file](https://github.com/mozilla/bedrock/blob/master/bedrock/careers/templates/careers/internships.html#L47)

Reply:
This is already what the `high_res_img` helper does, if you take a peek at the generated code. The internships file you link to is code copy/pasted from a different repo, so at the time it was written it didn't have the helper available. When we come to redesign the careers pages, we should use the common bedrock helpers.

Reply:
TIL! Thanks for the insight, you can disregard this then, Nathan ✨🪐